gradle.properties

```
@@ -1,5 +1,5 @@
 org.gradle.daemon=true
 
-selenium.version=4.9.1
+selenium.version=4.14.0
 # Please increment the value in a release
 appiumClient.version=8.6.0
```

```suggestion
appiumClient.version=9.0.0-SNAPSHOT
```
does it make sense to reflect the major version incremenet?

---

no, this should only be changed as the part of the release procudure